api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -192,6 +201,38 @@ message SendSatelliteCommandsRequest {
   repeated bytes command = 2;
 }
 
+// A request to modify configuration of ground station hardware.
+//
+// Next ID: 2
+// Status: ALPHA This API is under development and may not work correctly or be changed in backwards
+//         incompatible ways in the future.
+message GroundStationConfigurationRequest {
+  // A request to modify transmitter configuration at a ground station.
+  TransmitterConfigurationRequest transmitter_configuration_request = 1;
+}
+
+// A request to configure a transmitter at a ground station. Default state of these parameters
+// depends on hardware configuration at each ground station. Please check TransmitterConfiguration
+// in monitoring message. (Currently, some parameters are not supported.)
+// It isn't always supported to control them. If field isn't set, it's ignored.
+//
+// Next ID: 5
+// Status: ALPHA This API is under development and may not work correctly or be changed in backwards
+//         incompatible ways in the future.
+message TransmitterConfigurationRequest {
+  //  Whether the carrier is enabled.
+  google.protobuf.BoolValue enable_carrier = 1;
+
+  // Enable IF modulation
```

Comment:
These `BoolValue` comments should be consistent (either "Whether..." or "Enable...") and finish with a `.`.